render/render.js

```diff
@@ -780,9 +780,9 @@ module.exports = function($window) {
 		if(typeof input === "object") {
 			var style = {}
 			for(var key in input) {
-				style[key] = String(input[key]).replace(/[A-Z]/g, function(capital){
-					return "-" + capital.toLowerCase()
-				})
+				style[key] = key.charAt(0) === "-" && key.charAt(1) === "-"
+					? String(input[key])
+					: String(input[key]).replace(/[A-Z]/g, function(capital){ return "-" + capital.toLowerCase() })
```

Comment:
Perhaps a general purpose function for camelCase -> kebab-case? Where would something like that reside?

Reply:
You could just name it `toLowerCase` and drop it in the surrounding global-ish closure just above the function.

And I was referring to the `function(capital) { return "-" + capital.toLowercase() }` itself. The `.replace` call is fine where it is, I just want to lift the lambda out to avoid a perf cliff on setting styles.

Reply:
Ok :)

Out of curiosity. Would there be a performance change in pulling out the regex as well?

Reply:
Yes, a mild one (but not nearly as much as the function). You could pull that out, too, if you'd like.

Reply:
Alright :) And it's fine they're just lying there above updateStyle?

Reply:
Yep.

Reply:
perfect :)